src/main/resources/story/web/login.story

```diff
@@ -0,0 +1,19 @@
+Scenario: Log-in with invalid data
+Given I am on the main application page
```

Comment:
Please skip Given I am on the main application page as far as it is already a part of composite step